### PR TITLE
develop open, close

### PR DESCRIPTION
**Pull request by persona0220**
- fix sujin's ID at ".pullapprove.yml" file.
- change command & option code
- develop open
- develop close

### CODE REVIEW

File: src/js/manager.js [hunk 1]
```diff
@@ -3,30 +3,112 @@
...
+  }
 
   // Create a new tab with the url.
-  chrome.tabs.create({ "url": url, "selected": true });
+  chrome.tabs.create({"url": url, "selected": true});
 }
 
 function changeOption() {
-  let option = document.querySelector('#option');
+  var option = document.querySelector('#option');
```
Comment (kayla-kwon) [inline src/js/manager.js]: Use jQuery

File: src/js/manager.js [hunk 2]
```diff
@@ -3,30 +3,112 @@
...
+
+  var op = new Option();
+  for(var i=1; i<option.options.length; i++){
```
Comment (kayla-kwon) [inline src/js/manager.js]: You can use `.empty()` instead of removing each of them.

File: src/js/manager.js [hunk 3]
```diff
@@ -3,30 +3,112 @@
...
+      option.add(new Option('-url', '-url'));
+      option.add(new Option('-title', '-title'));
+      break;
+  }
+}
 
-  switch(document.querySelector('#command').value) {
+function commandSubmit(){
+  var option = document.querySelector('#option');
+  switch(document.querySelector('#command').value){
```
Comment (kayla-kwon) [inline src/js/manager.js]: Use jQuery

File: src/js/manager.js [hunk 4]
```diff
@@ -3,30 +3,112 @@
...
+            for(var i=0; i<tabs.length; i++){
+              if(tabs[i].url.indexOf(keyword) > -1){
+                chrome.tabs.remove(tabs[i].id);
+              }
+            }
+          });
+        }
+        else if (option.value === '-title') {
+          chrome.tabs.query({"currentWindow": true}, function (tabs) {
+            for(var i=0; i<tabs.length; i++){
```
Comment (kayla-kwon) [inline src/js/manager.js]: Follow the Google's code style.

**Review by kayla-kwon — DISMISSED** [review]

File: src/js/manager.js (hunk 1 above)
Comment (jurampark) [inline src/js/manager.js]: done.

**Review by jurampark — APPROVED** [review]
lgtm, thanks!

**Pull request merged**